pkg/services/ngalert/remote/client/alertmanager_configuration.go

```diff
@@ -17,24 +17,26 @@ const (
 )
 
 type GrafanaAlertmanagerConfig struct {
+	// TODO this needs to be deleted once Mimir is updated
```

Comment:
NIT: Can we leave a short "why" here for some quick context

Reply:
I have couple of prs in that area, I will update there